lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -5524,6 +5524,48 @@ class ConvertAtenOpToTosaCustomOp : public OpConversionPattern<AtenOpT> {
   std::string implementedWithOpAttr;
 };
 
+template <>
+class ConvertAtenOpToTosaCustomOp<AtenGeluOp>
+    : public OpConversionPattern<AtenGeluOp> {
+public:
+  using OpConversionPattern<AtenGeluOp>::OpConversionPattern;
+  using OpAdaptor = typename AtenGeluOp::Adaptor;
+
+  ConvertAtenOpToTosaCustomOp(TypeConverter &typeConverter,
+                              MLIRContext *context, std::string /*opName*/,
+                              std::string /*implementedWithOpAttr*/)
+      : OpConversionPattern<AtenGeluOp>(typeConverter, context) {}
+
+  LogicalResult
+  matchAndRewrite(AtenGeluOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+
+    std::string approximate;
+    if (!matchPattern(op.getApproximate(), m_TorchConstantStr(approximate)) ||
+        approximate != "none") {
```

Comment:
Why do we care whether approximate is `none` here? Can we forward all values?

Reply:
You are right. Removing it.

Reply:
That came from the original conversion, and I hadn't thought it through. Nice catch!